pkg/util/iptables/iptables_linux.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux
```

Comment:
why add this comment line?

Reply:
Since go 1.17, it will transition from // +build: to //go:build to do conditional compilation. Thus, when make lint with go 1.17 or later version, it will end in gofmt error.
More Info: https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md
I think we'd better to allow developers to use go 1.17.